integration-tests/oidc/src/main/resources/application.properties

```diff
@@ -14,7 +14,6 @@ quarkus.native.additional-build-args=-H:IncludeResources=.*\\.jks
 quarkus.http.cors=true
 quarkus.http.cors.origins=*
 
```

Comment:
See the OIDC code flow related comment, I propose to have a dedicated test in the oidc extension. Lets say one where the basic is set in the HTTP policy path and the bearer token is used, another one - with BasicAuthentication and also the bearer token being used

Reply:
> See the OIDC code flow related comment, I propose to have a dedicated test in the oidc extension.

In this particular test, I don't find the comment reasonable because it has zero impact on OIDC code flow.

> Lets say one where the basic is set in the HTTP policy path and the bearer token is used, another one - with BasicAuthentication and also the bearer token being used

really, here, I'm just dropping `quarkus.http.auth.basic=true` because there is a `quarkus.http.auth.permission.basic.auth-mechanism=basic`, I don't get it why it concerns you. Maybe you confused it with the `integration-tests/oidc-code-flow` where I did that refactoring?

Reply:
Anyway, even though I disagree I'll drop this change.

Reply:
@michalvavrik Sorry I missed it, I did not realize the basic mechanism was enabled at the policy level